hedera-node/hedera-app-spi/src/testFixtures/java/com/hedera/node/app/spi/fixtures/state/MapReadableKVState.java

```diff
@@ -37,7 +37,7 @@
  * @param <K> The key type
  * @param <V> The value type
  */
-public class MapReadableKVState<K extends Comparable<K>, V> extends ReadableKVStateBase<K, V> {
+public class MapReadableKVState<K extends Comparable<? super K>, V> extends ReadableKVStateBase<K, V> {
```

Comment:
Shouldn't need the `? super` anymore, it was there for Comparable, which is no longer required.

Reply:
I thought Artem's PR did not make it into the platform sdk `0.37.0-adhoc.xc76224af`. Will remove when next platform sdk is released

Reply:
Ahh, missed that this is still a little behind platform.
Yes, that would be a separate PR, then.